README.md

```diff
@@ -1,2 +1,71 @@
 # docker-itchysats
-ItchySats in a Docker container
+
+A repo facilitating running ItchySats inside a [Docker][https://www.docker.com/what-docker] container on a x86-64 (e.g. macOS, Linux, Windows) or ARMv7 system.
+Docker website provides extensive tutorials on Docker installation, please refer there if you are new to Docker.
+
+The following guide assumes that you have Docker running, having [docker-compose][https://docs.docker.com/compose/install/] installed is also recommended.
+
+# Quickstart
+
+## Testnet
```

Comment:
I would put `Mainnet` first. I think it should be more present than `Testnet`.

Reply:
why? why not allow people try it out of testnet first? I find it important to have it this way for the demo purposes and figuring out what the application can do, before any money is used?